addons/escoria-core/game/core-scripts/esc_item.gd

```diff
@@ -1,4 +1,7 @@
-# ESCItem is a Sprite that defines an item, potentially interactive
+# ESCItem is a Sprite that defines an item, potentially interactive.
```

Comment:
Ah, I just see it's actually not a sprite. That suggests a visual representation. You still need to add an actual Sprite2D to it. We should change that bit as well while you're at it.

I'd suggest:
```suggestion
# ESCItem defines a potentially interactive item of the game.
```